hydro_deploy/core/src/gcp.rs

```diff
@@ -213,7 +213,7 @@ impl GcpComputeEngineHost {
 #[async_trait]
 impl Host for GcpComputeEngineHost {
     fn target_type(&self) -> HostTargetType {
-        HostTargetType::Linux
+        HostTargetType::Linux(crate::LinuxArchitecture::AARCH64)
```

Comment:
Hmm, we should probably default to x86_64, and also have a way for users to pass this in when constructing the host (until later when we can auto-infer based on the machine type).